### DISCUSSION TITLE
Losslesscut upload to Youtube problem.

### DISCUSSION THREAD

**Discussion by fatcatthegreat** [mifi/lossless-cut#1501] [Support]
How do I overcome the "unable to process" error from youtube when trying to upload a losslesscut generated .mp4 or .mov?  Conversion in LLC goes fine as far as I can tell.  A file with the file name, then a long sequence of numbers and the correct extension is generated.  File is added to the youtube upload page and processing begins then stops and I get the error.  I have tried all the suggested fixes. No joy. Help will be appreciated.

**Comment by mifi** [comment]
Hi. can you play the file locally in all video players?

**Reply by fatcatthegreat** [reply]
Thank you for your interest and help.

The lossless-cut generated .mov files play on Windows Media Player without a problem. I tried to attach the file but it is too large.

**Reply by mifi** [reply]
Is it a long video? Maybe the promlematic time point is sometime in the middle of the video.
